pypeit/par/pypeitpar.py

```diff
@@ -1321,12 +1321,16 @@ def __init__(self, ex_value=None, flux_value=None, nmaskedge=None,
         dtypes['coaddfile'] = str
         descr['coaddfile'] = 'Output filename'
 
-
-        # Version checking
-        defaults['chk_version'] = True
-        dtypes['chk_version'] = bool
-        descr['chk_version'] = 'If True enforce strict PypeIt version checking to ensure that spec1d*.fits files were created' \
-                               'with the current version of PypeIt'
+        # NOTE: Deprecated in favor of the parameter in ReduxPar!
```

Comment:
Should we just delete this? I don't mind keeping it around for a while for developers, but I think a quick search of this file will find where `chk_version` is now kept.